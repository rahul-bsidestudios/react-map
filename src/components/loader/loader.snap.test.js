import React from 'react';
import renderer from 'react-test-renderer';
import Loader from './index';

test('renders Loader component', () => {
  const tree = renderer.create(<Loader />).toJSON();
  expect(tree).toMatchSnapshot();
});