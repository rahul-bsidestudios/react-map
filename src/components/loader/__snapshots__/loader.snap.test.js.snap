// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Loader component 1`] = `
<div
  className="loading"
>
  Loadingâ€¦
</div>
`;
